Marlin/src/lcd/menu/menu_temperature.cpp

```diff
@@ -242,14 +242,14 @@ void menu_temperature() {
         EDIT_ITEM_FAST(percent, MSG_EXTRA_FAN_SPEED_1, &thermalManager.new_fan_speed[0], 3, 255);
       #endif
     #endif
-    #if HAS_FAN1 || (ENABLED(SINGLENOZZLE) && EXTRUDERS > 1)
+    #if HAS_FAN1 || (DISABLED(SINGLENOZZLE) && EXTRUDERS > 1)
```

Comment:
This condition doesn't get rid of the issue that `HAS_FAN1` is made irrelevant. I will look into the back-history of this code to see why it is shaped the way it is.

Reply:
the issue is #15711 

I don't understand "HAS_FAN1 is made irrelevant",
it is defined in src/inc/Conditionals_post.h as follow:
#define HAS_FAN1 (PIN_EXISTS(FAN1) && CONTROLLER_FAN_PIN != FAN1_PIN && E0_AUTO_FAN_PIN != FAN1_PIN && E1_AUTO_FAN_PIN != FAN1_PIN && E2_AUTO_FAN_PIN != FAN1_PIN && E3_AUTO_FAN_PIN != FAN1_PIN && E4_AUTO_FAN_PIN != FAN1_PIN && E5_AUTO_FAN_PIN != FAN1_PIN)

while SINGLENOZZLE and EXTRUDERS are user definable in Configuration.h